### DISCUSSION TITLE
[BUG] When deploying changes to Mirth Connect installations on two different servers behind a load balancer, the deployment only apply to one server, even though both connect to the same database.

### DISCUSSION THREAD

**Discussion by mvndr** [nextgenhealthcare/connect#6107] [Q&A]
**Describe the bug**

When deploying changes to Mirth Connect installations on two different servers behind a load balancer, the deployment only apply to one server, even though both connect to the same database.

**To Reproduce**

- Install separate Mirth Connect instances on two different servers.
- Configure both instances to connect to the same database.
- Make changes to a specific channel in one Mirth Connect instance.
- Deploy the updated channel.
- Observe the second Mirth Connect instance:
- The updated channel information appears with last modified date, but the changes are not functionally applied because no deployment happen to other channel 
- Processing continues using the old channel configuration.

**Expected behavior**
Both Mirth Connect instances should reflect and implement the deployed changes simultaneously.

**Actual behavior**
Only the instance where the deployment originated receives the update. The other instance appears updated but functions with the previous configuration.

**Environment (please complete the following information):**
 - OS: Linux (CentOS)
 - Connect Version e.g. 4.4.1

**Workaround(s)**
 Need to individual deployed both mirth channels.

**Comment by kirbykn2** [comment]
Why are you expecting this?  Are you using the Advanced Clustering plugin?
"Both Mirth Connect instances should reflect and implement the deployed changes simultaneously."

**Comment by mvndr** [comment]
Hi @kirbykn2  , I am not using the Advanced Clustering plugin. Instead, I am primarily relying on a load balancer with a single database. The issue arises when changes are made in one instance connected to the same database as another instance. While changes can be reflected initially, deploying those changes does not synchronize them across both instances. Although the last modified date is visible in the channel history, the deployment history remains outdated.

**Comment by kirbykn2** [comment]
Deploying channels happens at the MC server.  You can share a backend database and have one server that does not have any deployed channels.  You need to redeploy the channels on the servers you want them redeployed too.

**Comment by pacmano1** [comment]
you should go over to the mirth forums site and read up on the things you need to manage yourself if you do not use the advanced custom plug-in.  For example https://forums.mirthproject.io/forum/mirth-connect/support/18952-horizontal-scaling-and-shared-database?p=104618#post104618.

**Comment by pacmano1** [comment]
This is not a supported configuration and therefore not an issue. I’m moving your issue to the discussion area.
